### ISSUE TITLE
'Links do not have a discernible name' social icons accessibility issue

### ISSUE THREAD

**Issue by MarinescuLucia** [Codeinwp/themeisle-companion#736]
Lighthouse throws a warning regarding links that don't have a discernible name which points to the social icons added with Orbit Fox.

Steps to replicate the issue:
1. Activate the Social Sharing module
2. Add some icons on a page
3. Generate the Lighthouse report and check the 'Links do not have a discernible name' accessibility failed audit

![Capture1](https://user-images.githubusercontent.com/86592341/158075941-da79549f-7811-4812-aa30-c0a5097367cb.PNG)

Reported here --> https://wordpress.org/support/topic/accessibility-problem-3/

**Comment by preda-bogdan** [comment]
The template is for social sharing module is missing an `aria-label` attribute [here](https://github.com/Codeinwp/themeisle-companion/blob/c99b3cc5b89dc4e2f73fc7f3fcb2596e420e353e/obfx_modules/social-sharing/views/social-sharing-tpl.php#L39-L51) and [here](https://github.com/Codeinwp/themeisle-companion/blob/c99b3cc5b89dc4e2f73fc7f3fcb2596e420e353e/obfx_modules/social-sharing/views/hestia-social-sharing-tpl.php#L38-L44)

This should fix the issue.

@arinaturcu if you have the time, it is not urgent but this could be a quick fix. I will assign this to you.
